General low hanging fruit for production (#227)
DigitalExcellence/dex-frontend: CHANGELOG.md

- Fixed issue where the alert container would display itself while there are no alerts active - [#200](https://github.com/DigitalExcellence/dex-frontend/issues/200)
- Fixed issue where user endpoint would get called when a user is not authenticated - [#189](https://github.com/DigitalExcellence/dex-frontend/issues/189)
- Fixed issue where empty search would cause a 404 error to appear on the projects overview page - [#179](https://github.com/DigitalExcellence/dex-frontend/issues/179)

> - Fixed users being able to open project-add modals even when they are not signed in - [#224](https://github.com/DigitalExcellence/dex-frontend/issues/224)

Reviewer: ```suggestion
- Fixed issue where a not authenticated user was able to open the project-add modals - [#224](https://github.com/DigitalExcellence/dex-frontend/issues/224)
```

Author: Appreciate it
